cmd/jaeger/internal/extension/jaegerquery/server.go

```diff
@@ -135,8 +138,8 @@ func (s *server) Shutdown(ctx context.Context) error {
 	if s.server != nil {
 		errs = append(errs, s.server.Close())
 	}
-	if s.jtracer != nil {
-		errs = append(errs, s.jtracer.Close(ctx))
+	if s.server.TracerProvider != nil {
```

Comment:
```suggestion
	if s.server.closeTracer != nil {
```
Why couple unnecessarily. Alternatively, you can jus put the code how it was, keep jtracer reference until we remove it completely.